src/renderers/WebGLRenderer.js

```diff
@@ -846,8 +846,22 @@ THREE.WebGLRenderer = function ( parameters ) {
 				renderer.setMode( _gl.LINES );
 
 			} else {
+				
+				switch ( object.mode ) {
 
-				renderer.setMode( _gl.TRIANGLES );
+					case THREE.TrianglePiecesDrawMode:
```

Comment:
Last one... Wouldn't it be better `THREE.TrianglesDrawMode` instead of `THREE.TrianglePiecesDrawMode`?